### DISCUSSION TITLE
Add features in Nemoguardrails to make the configuration of the bot configurable during runtime

### DISCUSSION THREAD

**Discussion by Shridhar2205** [NVIDIA-NeMo/Guardrails#652] [Ideas]
I have currently deployed a Nemo Guardrails server on a K8S pod using a Docker image. I am using my own custom class for the LLM model and have set an input and an output rail. 
Currently there is no such feature that allows you to enable/disable the rails that you have set up during runtime. In my case, that would mean that I would be able to specify that which rail (input/output) I want to enable for a particular curl request that is going to hit my NemoGuard server.
To achieve such a functionality, I would have to change the code each time in the config.yml and similar changes in other files of the config folder (specify the configuration of each rail and modify it again and again for each request), then build a Docker image for the updated code, push it again to K8S and then deploy it before testing.

If there can be a feature which would allow me to pass which rails to activate (or to modify similar configuration) while hitting that server (probably as an environment variable), it would save a lot of effort and time (and resources!!)

**Comment by drazvan** [comment]
Agree. Noted. We'll keep you posted.
